app/presenters/concerns/hyrax/iiif_av/displays_content_decorator.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Hyrax
+  module IiifAv
+    # main reasons for this decorator is to override variable names from hyrax-iiif_av
+    #   solr_document => object
+    #   current_ability => @ability
+    #   request.base_url => hostname
+    # also to remove #auth_service since it was not working for now
+    module DisplaysContentDecorator
+      private
+
+        def solr_document
+          defined?(super) ? super : object
+        end
+
+        def current_ability
+          defined?(super) ? super : @ability
+        end
+
+        Request = Struct.new(:base_url, keyword_init: true)
+
+        def request
+          Request.new(base_url: hostname)
+        end
+
+        def image_content
+          return nil unless latest_file_id
+          url = Hyrax.config.iiif_image_url_builder.call(
+            latest_file_id,
+            request.base_url,
+            Hyrax.config.iiif_image_size_default,
+            object.mime_type
+          )
+
+          # Serving up only prezi 3
+          image_content_v3(url)
+        end
+
+        def video_display_content(_url, label = '')
+          width = Array(solr_document.width).first.try(:to_i) || 320
```

Comment:
is there a reason we're wrapping a few values in this file in an array and then calling `.first`, instead of just checking for the presence of the value?

e.g.
``` ruby
# as an array
width = Array(solr_document.width).first.try(:to_i) || 320

# not as an array
width = solr_document.width&.try(:to_i) || 320
```

Reply:
made suggested changes and tested aspect ratio of video uploads